.github/workflows/check-backport-changelog.yml

```diff
@@ -31,6 +31,7 @@ jobs:
             - name: 'Fetch relevant history from origin'
               run: git fetch origin ${{ github.event.pull_request.base.ref }}
             - name: Check CHANGELOG status
+              if: ${{ !contains(github.event.pull_request.labels.*.name, 'No Core sync required') }}
```

Comment:
This could easily be "Backport from WordPress Core", but there might be other scenarios where a backport isn't required, e.g., adding `gutenberg` text domains to i18n translations.

Reply:
Good call. I'll test.

Reply:
Done